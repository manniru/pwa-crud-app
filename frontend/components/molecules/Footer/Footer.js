import * as sc from './Footer.style';

const Footer = () => (
  <sc.Footerr>
    <p>hmmStart</p>
  </sc.Footerr>
);

export default Footer;
