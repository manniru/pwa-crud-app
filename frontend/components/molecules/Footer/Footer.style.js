import styled from 'styled-components';

export const Footerr = styled.div`
  text-align: center;
`;
