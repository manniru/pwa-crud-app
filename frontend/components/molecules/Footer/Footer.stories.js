import Footer from './Footer';

export default { title: 'Components|Molecules', component: Footer };

export const footer = () => <Footer />;
