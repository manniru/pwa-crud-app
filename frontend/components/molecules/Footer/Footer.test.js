import { render, cleanup } from '@testing-library/react';
import { ThemeProvider } from 'styled-components';
import Footer from './Footer';
import theme from '../../../public/styles/theme.style';

const setup = (updatedProps = {}) => {
  const initialProps = {};
  const mergedProps = { ...initialProps, ...updatedProps };

  const result = render(
    <ThemeProvider theme={theme}>
      <Footer {...mergedProps} />
    </ThemeProvider>
  );

  const footer = () => result.queryByText('hmmStart');

  return { ...result, footer };
};

describe('Footer', () => {
  afterEach(cleanup);

  it('matches snapshot', () => {
    const utils = setup();

    expect(utils.baseElement).toMatchSnapshot();
  });

  it('renders elements', () => {
    const utils = setup();

    expect(utils.footer()).toBeInTheDocument();
  });
});
