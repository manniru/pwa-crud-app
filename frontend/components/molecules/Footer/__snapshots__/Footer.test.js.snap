// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer matches snapshot 1`] = `
.c0 {
  text-align: center;
}

<body>
  <div>
    <div
      class="c0"
    >
      <p>
        NGS
      </p>
    </div>
  </div>
</body>
`;
